webpush/realpush_test.go

```diff
@@ -0,0 +1,51 @@
+package webpush
```

Comment:
I have a very similar test script on my own machine that I didn't check in, to avoid putting any real subscriptions in the codebase. There is a very minor information leak (especially since mine has my GCM auth key in it) and the details basically only work for a single user, because a full integration test means manually checking that you received a notification.

@gauntface, what is the state of the art for testing push with real browsers? I feel like the node module has something better.

All that said, I guess I'd rather have imperfect integration tests than none.